riak/transports/pbc/connection.py

```diff
@@ -165,7 +165,7 @@ def _ssl_handshake(self):
 
                     return True
                 except ssl.SSLError as e:
-                    raise SecurityError(e.library + ": " + e.reason)
```

Comment:
This is the output with python 2.7.9 and the change to pass the exception directly to the `SecurityError` ctor:

```python
  File "/home/lbakken/Projects/basho/riak-python-client/riak/transports/pbc/connection.py", line 84, in _init_security
    self._ssl_handshake()
  File "/home/lbakken/Projects/basho/riak-python-client/riak/transports/pbc/connection.py", line 168, in _ssl_handshake
    raise SecurityError(e)
SecurityError: SSLEOFError(8, u'EOF occurred in violation of protocol (_ssl.c:581)')
```

This is what happens with the original code:

```python
  File "/home/lbakken/Projects/basho/riak-python-client/riak/transports/pbc/connection.py", line 84, in _init_security
    self._ssl_handshake()
  File "/home/lbakken/Projects/basho/riak-python-client/riak/transports/pbc/connection.py", line 169, in _ssl_handshake
    raise SecurityError(e.library + ": " + e.reason)
TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'
```